examples/mongodb/0.9/Dockerfile

```diff
@@ -5,4 +5,8 @@ MAINTAINER Debezium Community
 COPY init-inventory.sh /usr/local/bin/
 RUN chmod +x /usr/local/bin/init-inventory.sh
 
-CMD ["mongod", "--replSet", "rs0", "--auth"]
+COPY init-inventory-replicaset.sh /usr/local/bin/
+RUN chmod +x /usr/local/bin/init-inventory-replicaset.sh
+
+# CMD ["mongod", "--replSet", "rs0", "--auth"]
+CMD ["mongod", "--replSet", "rs0"]
```

Comment:
@jpechane, any idea how to make the "--auth" parameter optional, depending on how an env var passed when starting a container from that image? I could create a separate file of course but really hoped it can be avoided, so the example image can be use clustered and non-clustered. Thanks!

Reply:
How about introducing a script that will start the mongo and use it in `CMD` instruction? The script can then easily add/remove options as needed?